docs/logging.md

```diff
@@ -0,0 +1,67 @@
+Include a logger in your module with:
+
+```javascript
+var logger = require('react-server').logging.getLogger(__LOGGER__);
+```
+
+This `logger` has methods for many log levels. In order: emergency, alert,
+critical, error, warning, notice, info, debug.
```

Comment:
Might be nice to have these in list format to make them jump out at someone who's scanning for them.